test/recipe/admin/pages/specs/locale.spec.js

```diff
@@ -1,7 +1,7 @@
 import { expect, test } from "@nuxt/test-utils/playwright";
 import primevue from "~/utils/locators/primevue";
 
-test.beforeEach(async({goto}) => {
+test.beforeEach(async({ goto }) => {
```

Comment:
remove these whitespaces `({ goto })`, i know it doesn't look good, but for the sake of consistency leave it non-spaced.